templates/deployment.yaml

```diff
@@ -54,6 +54,11 @@ spec:
           - /etc/docker/registry/config.yml
           ports:
             - containerPort: 5000
+{{- if .Values.configData.http.debug.prometheus.enabled }}
```

Comment:
This is a deeply nested configuration path, increasing the risk that some ancestor along the way is nil. Is there a "maybe" syntax or helper function that can be used here to short-circuit to false in that case?

Reply:
he can continue, I close this